GVRf/Framework/jni/shaders/posteffect/custom_post_effect_shader.cpp

```diff
@@ -137,6 +139,16 @@ void CustomPostEffectShader::render(
         glUniform1i(u_texture_, texture_index++);
     }
 
+    if (u_projection_matrix_ != -1) {
+        glm::mat4 view = camera->getViewMatrix();
+        glm::mat4 proj = glm::mat4(
```

Comment:
the rest of the code looks pretty straight-forward, this line is the only one I have a question on so far.  Why this extra step of creating another mat4?